source/release-notes/1.10.txt

```diff
@@ -17,4 +17,11 @@ Release Notes for mongosync 1.10
 .. _1.10.0-c2c-release-notes:
 
 This page describes changes and new features introduced in  
-{+c2c-full-product-name+} 1.10 and the {+c2c-full-beta-program+}.
\ No newline at end of file
+{+c2c-full-product-name+} 1.10 and the {+c2c-full-beta-program+}.
```

Comment:
could you provide a staging link for this page too?

Reply:
added to the description.